### DISCUSSION TITLE
글자 색상 관련

### DISCUSSION THREAD

**Discussion by koomchang** [pronist/hELLO#209] [Q&A] [answered]
다크모드 이고 블로그 글을 쓰면 정말 임의로? 일부 글자색이 검정색으로 보입니다. 다 그런 것도 아니고 일부만 그래서 어디가 문제인지 몰라 html 모드에서 일일히 수정해주는데 이걸 어떻게 설정해야 모두 흰색 글씨로 잘 나오게 할 수 있을가요?

<p data-ke-size="size16"> 이런 태그 안에 제가 원하지도 않은 color 속성이 가끔 들어갑니다 ㅠㅠ

**Comment by pronist** [comment, marked as answer]
@koomchang 안녕하세요.

일부 글자색이 검은색으로 보이는 경우, 열에 아홉은 다른 곳에서 복사&붙여넣기로 글을 쓴 경우에 발생합니다. 또는 마크다운으로 글을 작성했을 때도 나타날 수 있습니다. 다른 플랫폼에서 티스토리 에디터로 글을 복붙하는 경우 티스토리 에디터에서 자체적으로 인라인으로 색상을 부여하는 등 블로거가 원하지 않은 행동을 하게 됩니다. 반면, 색상을 강제적으로 스킨에서 부여하면 블로거의 자유도를 침해하는 행동이 될 수도 있습니다.

이 문제를 해결하기 위해서는 블로거가 복사&붙여넣기 또는 마크다운으로 쓰는 것을 자제하고 티스토리 기본 에디터로 작성하는 것을 기본으로 하셔야 합니다.

**Reply by koomchang** [reply]
친절한 답변 감사드립니다!!
